client/packages/host/webpack.config.js

```diff
@@ -99,7 +99,6 @@ module.exports = env => {
       new CopyPlugin({
         patterns: [
           { from: './public/mockServiceWorker.js', to: 'mockServiceWorker.js' },
-          { from: './public/config.js', to: 'config.js' },
```

Comment:
this is an older branch, right? you should update from main.
and when you do.. we'd have to rework the `start-local` scripts as well

Reply:
Yeah, I'll have to do some changes to this branch to allow for start-remote and start-local

Reply:
In this commit 7cfdfb5fa2f3058ad1ba5871c45a6daa5d9e06c1, I attempted to reinstate local and remote yarn scripts